modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/login-button/index.html

```diff
@@ -0,0 +1,3 @@
+<div class="align-items-end d-flex evp-login-button justify-content-center">
```

Comment:
Those classes removed, could be inside `.evp-login-button` css

```suggestion
<div class="evp-login-button">
```